style.css

```diff
@@ -0,0 +1,720 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* ------------------navbar section------------------- */
+.nav-container {
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+  align-items: center;
+}
+
+.nav-container a {
+  text-decoration: none;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  padding-left: 20px;
+}
+
+.nav-container .items-menu {
+  display: flex;
+  align-items: center;
+  margin-left: 700px;
+  flex: 1;
+  height: 100%;
+  color: white;
+  text-decoration: none;
+  list-style: none;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-weight: 700px;
+}
+
+.nav-container .items-menu li a {
+  margin-right: 40px;
+  color: white;
+  list-style: none;
+  text-decoration: none;
+  font-size: 18px;
+  height: 100%;
+  padding-left: 10px;
+  padding-right: 10px;
+  transition: ease-in-out 5s;
+}
+
+.nav-container .items-menu li a:hover {
+  background-color: #615f5f;
+}
+
+.nav-container a .fa-bars {
+  display: none;
+}
+
+/* ---------------------------------------hero section------------------------------------ */
+.hero-section .hero-row {
+  display: flex;
+  background-color: #1c1a19;
+  width: 1440px;
+  height: 500px;
+}
+
+.hero-left {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-left: 30px;
+  margin-top: 100px;
+  flex: 0.2;
+  height: 100px;
+  transition: transform 5s;
+  order: 1;
+}
+
+.hero-left a i {
+  background: #1c1a19;
+  color: #cfcfcf;
+}
+
+.hero-left a i:hover {
+  color: brown;
+  border-radius: 50px;
+}
+
+.hero-right {
+  display: flex;
+  flex-direction: column;
+  margin-left: 40px;
+  margin-top: 60px;
+  flex: 0.5;
+  height: 300px;
+  order: 2;
+}
+
+.hero-right h1 {
+  font-size: 26px;
+  color: rgb(155, 116, 19);
+}
+
+.hero-right h5 {
+  color: rgb(204, 194, 194);
+  font-size: 24px;
+  font-family: 'Roboto', sans-serif;
+}
+
+.hero-right p {
+  margin-top: 20px;
+  color: rgb(204, 194, 194);
+  font-size: 14px;
+}
+
```

Comment:
[OPTIONAL] You did a great job implementing the previous reviewer request but also fix the size and the color of your heading elements. Check the images below.
<img width="845" alt="Screenshot 2023-01-16 at 8 54 01 PM" src="https://user-images.githubusercontent.com/25887253/212741536-0bac34fb-dd92-43d1-b6c3-038d7aafea33.png">
<img width="845" alt="Screenshot 2023-01-16 at 8 56 22 PM" src="https://user-images.githubusercontent.com/25887253/212741538-f0c5d493-e24a-489b-87fa-3c4abc93742f.png">